oeps/processes/oep-0055-project-maintainers.rst

```diff
@@ -0,0 +1,309 @@
+
+Abstract
+========
+
+The Open edX platform is a service oriented platform for authoring and
+delivering online learning based on the `Django framework`_.  At the
+highest level, the platform is composed of numerous services (IDAs
+[#]_), micro-front ends (MFEs [#]_) and libraries.  The code for these
+services and libraries is managed in git and hosted on GitHub
+platform.
+
+Each component part of the Open edX platform requires maintenance to
+ensure that it meets, and continues to meet, the project definition of
+high quality.  That definition will include project standards,
+appropriate documentation, and fitness for use, among other things.
+That definition will evolve as project needs do and we learn what
+works and what doesn't.
+
+Component maintenance is work.  To ensure that that work is done, a
+person or persons must be recognized as responsible for doing it,
+and held accountable for its performance.
+
+The Open edX project, like many projects, calls this role
+"maintainer."  The notion of a project maintainer is typical of
+open-source projects and widely considered an best practice [#]_.
+
+The maintainer role is a leadership role that encompasses both
+technical and community elements.  Both dimensions of the role are
+equally important.
+
+All, non-deprecated, components of the Open edX project must have a
+named maintainer.  When no such maintainer exists, this will be
+considered an urgent problem to solve.
+
+This OEP proposes a formal definition of the role of maintainer that
+will open to all qualified Open edX community members.  Maintainers
+will be responsible for the health of platform components, but not for
+their performance in production [#]_.
+
+Motivation
+==========
+
+With the change in stewardship from edX to The Center for Reimagining
+Learner, it is essential to have a well defined plan for maintaining
+the platform.
+
+In order to ensure that the project is sustainable for the long term,
+maintenance must be shared by the entire community.  The project must
+be robust enough that a it can tolerate firms and contributors leaving
+the ecosystem.
+
+As a project we must acknowledge that maintenance is hard work, but
+work that must be done for the platform to be fit for use.  The best
+way of ensuring this is to have a named person or persons who are
+responsible for getting the work done.
+
+Specification
+=============
+
+Definitions
+-----------
+
+The term **repository** will mean a repository on the GitHub social
+coding platform.
+
+The term **component** will typically mean the unit of software housed
+in a single GitHub repository.  Exceptions will be made in the case of
+edx-platform where there may be multiple distinguishable components in
+that single repository.  Components will typically, but not always, be
+hosted at the Open edX GitHub organization.  Components will be "part
+of" or "critical to" the open-source platform.
```

Comment:
Yes.  I was thinking roughly that something like the Hooks and Signal API is "part of" the platform, while something like Tutor is now "critical to" the platform as it's our default deployment artifact.  Let me know if you think this distinction is misplaced, wrong, incomplete, confusing, all of the above, etc.

Reply:
It makes more sense when you explain it, but I'm not sure how to clarify. Could it be the distinction that "part of" mean part of the default deployment, and "critical to" means tools and supporting code/configuration that are needed in order to run the deployment? I don't think that's quite right though...

Reply:
@nedbat can you please review and let me know if this can be resolved?  Here you requested changes, so I want to ensure the concerns have been addressed.

Reply:
I think it's a fine distinction. I didn't know if you meant "or" as two alternatives, or two synonyms.  Just make it:
```
Components will be either "part of" or "critical to" the open source platform.
```